scripts/clan_package/settings/clan_settings.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import ujson
+
+from scripts.game_structure.game.save_load import safe_save
+from scripts.game_structure.game.switches import Switches, get_switch
+from scripts.housekeeping.datadir import get_save_dir
+
+
+def load_clan_settings():
+    reset_loaded_clan_settings()
+    if os.path.exists(
+        get_save_dir() + f"/{get_switch(Switches.clan_list)[0]}/clan_settings.json"
+    ):
+        with open(
+            get_save_dir() + f"/{get_switch(Switches.clan_list)[0]}/clan_settings.json",
+            "r",
+            encoding="utf-8",
+        ) as write_file:
+            _load_settings = ujson.loads(write_file.read())
+
+        for key, value in _load_settings.items():
+            if key in clan_settings:
+                clan_settings[key] = value
+
+    # if settings files does not exist, default has been loaded by __init__
+
+
+def save_clan_settings():
+    safe_save(
+        get_save_dir() + f"/{get_switch(Switches.clan_name)}/clan_settings.json",
```

Comment:
Moving forward, we should try to use Pathlib instead of constructing path literals as strings.